### DISCUSSION TITLE
Lovely Fright Whilst Maintaining my Server

### DISCUSSION THREAD

**Discussion by toydotgame** [ObscenityIB/creeper#6] [General]
I already had a whitelist, so I'm relatively fine.

I know this project is made with good intentions in mind, but I don't know much about the specific person running it and what they are going to do.

I think I'll need to do something at one point because that one ping on TCP `25565` is not much, but I checked my HFS log and I've got tons of pings on TCP `80`. (About 1 every hour)
Time to do some server admin work! _Woo..._

**Comment by ObscenityIB** [comment]
Oh yeah, everyone gets scanned and probed constantly, mostly from china, that's just the nature of the internet.
Even without any malicious intent, there are still constant scans for every reason you could think of, search engines, DNS, proxy lists, crazy live maps of a 3D rendered earth with traffic flow animated across it....

It's never a quiet day for the information highway.

**Comment by ObscenityIB** [comment]
As for what I'm going to do, the purpose of building this was to find random servers to join, really quickly.
Once I got to a few people that were _outraged_ that I _dare_ step into a _public_ space, I made this public on GitHub and added a script that would let everyone whose server it joined know, **public** _means_ **public**, and at least basic networking should be known.

I've seen examples of some people who have so little of a clue they consider a ping "hacking" or "attacking" them.
Imagine how many thousands of abuse emails that person must send 😆 

But yeah, so far it only does Telstra, Microplex, TPG, Vodafone, and Foxtel in AU, and AT&T and Comcast in the US from my server.
Everything is logged if I need it later, like for statistics or something.

Speaking of, were you online at the time? I could look up your IGN and see what was done, I'm guessing you just got a single incomplete tcp session from the scan, then a bit later a ping from mcc trying to join, then some logs in minecraft related to that join attempt.
Since you said you have a whitelist already, it wouldn't have spoken in chat.

**Reply by toydotgame** [reply]
I don't believe anyone was online at the time, and all I saw of this script were these messages in the log:
```
18:43:34 [INFO] /x.x.x.x.:34344 lost connection
19:17:50 [INFO] Disconnecting creeper_dot_sh [/x.x.x.x:35256]: You are not whitelisted on ICCMC!
```
ISP is Telstra, Spigot server on 1.6.4 on Win7 ThinkCentre PC at home. At this point it's not helpful, but my IGN is `Toydotgam`, and the only other player that day was `Redo122`.

Interestingly, a WHOIS lookup gives this: (Yes, IPs for all to see!)
```
OVH Hosting, Inc. HO-2 (NET-139-99-0-0-1) 139.99.0.0 - 139.99.255.255
OVH Australia PTY LTD OVH-AU-1 (NET-139-99-128-0-1) 139.99.128.0 - 139.99.255.255
OVH Australia PTY LTD VPS-SYD (NET-139-99-200-0-1) 139.99.200.0 - 139.99.203.255
```
It's an Australian IP. Strange. It's not one of Telstra or Vodafone, who are the two ISPs my friends connect to this server with.

(Also, Foxtel is a service of Telstra; like Fetch is a service of Optus - so, what's Foxtel doing on that list?)

**Reply by ObscenityIB** [reply]
Foxtel also [provide NBN through Telstra](https://www.foxtel.com.au/shop/broadband-bundles.html), so they are kind of bundled together, but are also distinct.

Right now I'm using my old minecraft server to run the script, I'm thinking of spinning up another vm just for this, that way I can point everything back to the right place, reverse dns, redirect, etc

Just hoping GitHub pages doesnt complain if I add another A record that isn't githubs.
Something to try.

**Reply by toydotgame** [reply]
Foxtel seem to have products for nbn, but I'm quite sure the actual ISP behind this would be their parent, Telstra. :)

Bit confused on what you mean by the second paragraph, but then again - I've only read the README. I should see how this thing works under the hood later...
(At this point, clueless on the 3rd paragraph...)

Aside from this, I should definitely set another port other than 80 for my HTTP server, because it's very open right now.

I've got to sleep at some point as well (GMT+10, it's 11:30pm), so I'll be back tomorrow! ;)
